File: iPad/VerBank-CSTSv3-IOSClientApi/CSTS3Comm/comm/UserLogin.m
//
//  UserLogin.m
//  VerBank-CSTSv3-IOSClientApi
//
//  Created By Zhanglei on 15/6/30.
//  Copyright (c) 2015年 allone. All rights reserved.
//

#import "UserLogin.h"
static NSString * jsonId = @"UserLogin";

static NSString * aeid = @"1";
static NSString * type = @"2";
static NSString * realName = @"3";
static NSString * userName = @"4";
static NSString * callEmail = @"5";
static NSString * callPhone = @"6";
static NSString * marginCallWay = @"7";
static NSString * reckoning = @"8";
static NSString * reckoningType = @"9";
static NSString * disabled = @"10";

static NSString * ipFixed = @"11";
static NSString * ipList = @"12";
static NSString * regestTime = @"13";
static NSString * regestDealer = @"14";
static NSString * expireDate = @"15";
static NSString * messageRecType = @"16";
static NSString * msgLocal = @"17";
static NSString * firstLoginTime = @"18";
static NSString * lastLoginTime = @"19";

static NSString * maxTradeAmount = @"20";
static NSString * StmntLang = @"21";
static NSString * StmntFreq = @"22";
static NSString * DlyStmntMail = @"23";
static NSString * PeriodStmntMail = @"24";
static NSString * CnfrmMail = @"25";
static NSString * StmntOpenPsln = @"26";
static NSString * secretUser = @"27";
static NSString * caDownloadNumber = @"28";
static NSString * caLimitNumber = @"29";

@implementation UserLogin
- (id)init{
    self = [super init];
    if(self != nil)
    {
        [super setEntry:@"jsonId" entry:jsonId];
    }
    return self;
}

jsonImplementionString(Aeid,        aeid)
jsonImplementionInt(Type,           type)
jsonImplementionString(RealName,    realName)
jsonImplementionString(UserName,    userName)
jsonImplementionString(CallEmail,   callEmail)
jsonImplementionString(CallPhone,   callPhone)
jsonImplementionInt(MarginCallWay,  marginCallWay)
jsonImplementionInt(Reckoning,      reckoning)
jsonImplementionInt(ReckoningType,  reckoningType)
jsonImplementionInt(Disabled,       disabled)

jsonImplementionInt(IpFixed,        ipFixed)
jsonImplementionString(IpList,      ipList)
jsonImplementionDate(RegestTime ,   regestTime)
jsonImplementionString(RegestDealer,regestDealer)
jsonImplementionDate(ExpireDate,    expireDate)
jsonImplementionString(MessageRecType,messageRecType)
jsonImplementionString(MsgLocal,    msgLocal)
jsonImplementionDate(FirstLoginTime,firstLoginTime)
jsonImplementionDate(LastLoginTime, lastLoginTime)

jsonImplementionDouble(MaxTradeAmount,  maxTradeAmount)
jsonImplementionInt(StmntLang,          StmntLang)
jsonImplementionInt(StmntFreq,          StmntFreq)
jsonImplementionInt(DlyStmntMail ,      DlyStmntMail)
jsonImplementionInt(PeriodStmntMail ,   PeriodStmntMail)
jsonImplementionInt(CnfrmMail ,         CnfrmMail)
jsonImplementionInt(StmntOpenPsln ,     StmntOpenPsln)
jsonImplementionInt(SecretUser ,        secretUser)

jsonImplementionInt(CaDownloadNumber ,  caDownloadNumber)
jsonImplementionInt(CaLimitNumber,      caLimitNumber)

@end

File: WeChatService/EX02-CSTSv3-ClientAPI/src/jedi/ex01/CSTS3/comm/struct/UserLogin.java
package jedi.ex01.CSTS3.comm.struct;

import java.util.Date;

public class UserLogin extends allone.json.AbstractJsonData {

	public static final String jsonId = "26";

	public static final String aeid = "1";
	public static final String type = "2";
	public static final String realName = "3";
	public static final String userName = "4";
	public static final String callEmail = "5";
	public static final String callPhone = "6";
	public static final String marginCallWay = "7";
	public static final String reckoning = "8";
	public static final String reckoningType = "9";
	public static final String disabled = "10";
	public static final String maxOnlineNum = "11";
	public static final String ipFixed = "12";
	public static final String ipList = "13";
	public static final String regestTime = "14";
	public static final String regestDealer = "15";
	public static final String expireDate = "16";
	public static final String messageRecType = "17";
	public static final String msgLocal = "18";
	public static final String canLoginIBClient = "19";
	public static final String firstLoginTime = "20";
	public static final String lastLoginTime = "21";
	
	public static final String certificationType = "22";
	public static final String certificationId = "23";

	public UserLogin(){
		super();
		setEntry("jsonId", jsonId);
	}

	public String getAeid() {
		try {
			String data=getEntryString(UserLogin.aeid);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setAeid(String aeid) {
		setEntry(UserLogin.aeid, aeid);
	}

	public int getType() {
		try {
			int data=getEntryInt(UserLogin.type);
			return data;
		} catch (RuntimeException e) {
			return 0;
		}
	}

	public void setType(int type) {
		setEntry(UserLogin.type, type);
	}

	public String getRealName() {
		try {
			String data=getEntryString(UserLogin.realName);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setRealName(String realName) {
		setEntry(UserLogin.realName, realName);
	}

	public String getUserName() {
		try {
			String data=getEntryString(UserLogin.userName);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setUserName(String userName) {
		setEntry(UserLogin.userName, userName);
	}

	public String getCallEmail() {
		try {
			String data=getEntryString(UserLogin.callEmail);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setCallEmail(String callEmail) {
		setEntry(UserLogin.callEmail, callEmail);
	}

	public String getCallPhone() {
		try {
			String data=getEntryString(UserLogin.callPhone);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setCallPhone(String callPhone) {
		setEntry(UserLogin.callPhone, callPhone);
	}

	public int getMarginCallWay() {
		try {
			int data=getEntryInt(UserLogin.marginCallWay);
			return data;
		} catch (RuntimeException e) {
			return 0;
		}
	}

	public void setMarginCallWay(int marginCallWay) {
		setEntry(UserLogin.marginCallWay, marginCallWay);
	}

	public int getReckoning() {
		try {
			int data=getEntryInt(UserLogin.reckoning);
			return data;
		} catch (RuntimeException e) {
			return 0;
		}
	}

	public void setReckoning(int reckoning) {
		setEntry(UserLogin.reckoning, reckoning);
	}

	public int getReckoningType() {
		try {
			int data=getEntryInt(UserLogin.reckoningType);
			return data;
		} catch (RuntimeException e) {
			return 0;
		}
	}

	public void setReckoningType(int reckoningType) {
		setEntry(UserLogin.reckoningType, reckoningType);
	}

	public int getDisabled() {
		try {
			int data=getEntryInt(UserLogin.disabled);
			return data;
		} catch (RuntimeException e) {
			return 0;
		}
	}

	public void setDisabled(int disabled) {
		setEntry(UserLogin.disabled, disabled);
	}

	public int getMaxOnlineNum() {
		try {
			int data=getEntryInt(UserLogin.maxOnlineNum);
			return data;
		} catch (RuntimeException e) {
			return 0;
		}
	}

	public void setMaxOnlineNum(int maxOnlineNum) {
		setEntry(UserLogin.maxOnlineNum, maxOnlineNum);
	}

	public int getIpFixed() {
		try {
			int data=getEntryInt(UserLogin.ipFixed);
			return data;
		} catch (RuntimeException e) {
			return 0;
		}
	}

	public void setIpFixed(int ipFixed) {
		setEntry(UserLogin.ipFixed, ipFixed);
	}

	public String[] getIpList() {
		try {
			String[] data=getEntryObjectVec(UserLogin.ipList,new String[0]);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setIpList(String[] ipList) {
		setEntry(UserLogin.ipList, ipList);
	}

	public Date getRegestTime() {
		try {
			Date data=getEntryDate(UserLogin.regestTime);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setRegestTime(Date regestTime) {
		setEntry(UserLogin.regestTime, regestTime);
	}

	public String getRegestDealer() {
		try {
			String data=getEntryString(UserLogin.regestDealer);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setRegestDealer(String regestDealer) {
		setEntry(UserLogin.regestDealer, regestDealer);
	}

	public Date getExpireDate() {
		try {
			Date data=getEntryDate(UserLogin.expireDate);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setExpireDate(Date expireDate) {
		setEntry(UserLogin.expireDate, expireDate);
	}

	public String getMessageRecType() {
		try {
			String data=getEntryString(UserLogin.messageRecType);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setMessageRecType(String messageRecType) {
		setEntry(UserLogin.messageRecType, messageRecType);
	}

	public String getMsgLocal() {
		try {
			String data=getEntryString(UserLogin.msgLocal);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setMsgLocal(String msgLocal) {
		setEntry(UserLogin.msgLocal, msgLocal);
	}

	public int getCanLoginIBClient() {
		try {
			int data=getEntryInt(UserLogin.canLoginIBClient);
			return data;
		} catch (RuntimeException e) {
			return 0;
		}
	}

	public void setCanLoginIBClient(int canLoginIBClient) {
		setEntry(UserLogin.canLoginIBClient, canLoginIBClient);
	}

	public Date getFirstLoginTime() {
		try {
			Date data=getEntryDate(UserLogin.firstLoginTime);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setFirstLoginTime(Date firstLoginTime) {
		setEntry(UserLogin.firstLoginTime, firstLoginTime);
	}

	public Date getLastLoginTime() {
		try {
			Date data=getEntryDate(UserLogin.lastLoginTime);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setLastLoginTime(Date lastLoginTime) {
		setEntry(UserLogin.lastLoginTime, lastLoginTime);
	}


	public int getCertificationType() {
		try {
			int data=getEntryInt(UserLogin.certificationType);
			return data;
		} catch (RuntimeException e) {
			return 0;
		}
	}

	public void setCertificationType(int certificationType) {
		setEntry(UserLogin.certificationType, certificationType);
	}
	
	public String getCertificationId() {
		try {
			String data=getEntryString(UserLogin.certificationId);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setCertificationId(String certificationId) {
		setEntry(UserLogin.certificationId, certificationId);
	}
}